cli/cmd/internal/client/create_org.go

```diff
@@ -0,0 +1,47 @@
+package client
+
+import (
+	"code.cloudfoundry.org/quarks-utils/pkg/cmd"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/suse/carrier/cli/paas"
+)
+
+var ()
+
+// CmdCreateOrg implements the carrier orgs command
+var CmdCreateOrg = &cobra.Command{
+	Use:   "create-org [args]",
+	Short: "Creates an org",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		app, cleanup, err := paas.BuildApp(cmd.Flags(), nil)
+		defer func() {
+			if cleanup != nil {
+				cleanup()
+			}
+		}()
+
+		if err != nil {
+			return errors.Wrap(err, "error initializing cli")
+		}
+
+		err = app.CreateOrg(args[0])
+		if err != nil {
+			return errors.Wrap(err, "error creating org")
+		}
+
+		return nil
+	},
+	SilenceErrors: true,
+	SilenceUsage:  true,
+}
+
+func init() {
+	pf := CmdCreateOrg.PersistentFlags()
+
+	argToEnv := map[string]string{}
+
+	cmd.KubeConfigFlags(pf, argToEnv)
```

Comment:
From [the docs](https://github.com/spf13/cobra#persistent-flags):

```
A flag can be 'persistent', meaning that this flag will be available to the command it's assigned to as well as every command under that command. For global flags, assign a flag as a persistent flag on the root.
```
Given this method only adds `kubeconfig` et al (and nothing specific to this command or any subcommands), I think we should only do this on the root cmd, the probably in `main` (not `init`).